Source/WarpX.H

```diff
@@ -65,6 +65,8 @@
 #include <string>
 #include <vector>
 
+#include <openPMD/openPMD.hpp>
```

Comment:
Since no types or functions from openPMD are referenced in the header, it shouldn't be included here. Put the include in the relevant cpp files that make openPMD calls, or hpp files of classes that have members/function parameters of openPMD types.